pulpcore/app/tasks/orphan.py

```diff
@@ -71,8 +82,13 @@ def orphan_cleanup(content_pks=None, orphan_protection_time=settings.ORPHAN_PROT
         code="clean-up.artifacts",
     ) as progress_bar:
         for artifact in progress_bar.iter(artifacts.iterator()):
-            # we need to manually call delete() because it cleans up the file on the filesystem
-            artifact.delete()
+            try:
+                # we need to manually call delete() because it cleans up the file on the filesystem
+                artifact.delete()
+            except ProtectedError as e:
+                # Rarely artifact could be shared between to different content units.
```

Comment:
I would not even call this rare. The empty file is probably shared quite a lot. Also any package whatsoever may make sense to be in a file repository, and in the end, PublishedContent is some sort of Content too.
```suggestion
                # Rarely artifact could be shared between different content units.
```

Reply:
I took this from reclaim space, but this is wrong so I am going to adjust the message.